File: python/java_to_python/03-pgmPrintf/pgmPrintf.py
import datetime
import time

class pgmPrintf:
    def main(self):
        print('|----------------------------------|')
        print('|-------------PGM: printf----------|')
        print('|---Author:🐍 Santiago Duran 🐍----|')
        print('|----------------------------------|')

        try:
            print(f"Imprimiendo desde el shell a las {datetime.datetime.now()}⌚")
            time.sleep(3)
            print("📖📖📖📖📖📖📖📖📖")
            time.sleep(1)
            print("🖨️  🖨️  🖨️   Impresion finalizada 🖨️  🖨️  🖨️")
        except Exception as e:
            print(f"⚠️⚠️⚠️{e}")
            
pgmPrintf().main()

File: ruby/pgms_ruby/03-pgmPrintf/pgmPrintf.rb
require 'date'

class PgmPrintf
  def main
    puts '|----------------------------------|'
    puts '|-------------PGM: printf----------|'
    puts '|---Author:🐍 Santiago Duran 🐍----|'
    puts '|----------------------------------|'
    
    begin
      puts "Imprimiendo desde el shell a las #{Time.now}⌚"
      sleep(3)
      puts "📖📖📖📖📖📖📖📖📖"
      sleep(1)
      puts "🖨️ 🖨️ 🖨️   Impresion finalizada 🖨️ 🖨️ 🖨️"
    rescue => e
      puts "⚠️⚠️⚠️#{e}"
    end
  end
end

PgmPrintf.new.main